src/main/java/com/marklogic/kafka/connect/sink/DefaultSinkRecordConverter.java

```diff
@@ -50,6 +50,9 @@ public DocumentWriteOperation convert(SinkRecord sinkRecord) {
 	 * @return
 	 */
 	protected AbstractWriteHandle toContent(SinkRecord record) {
+		if ((record == null) || (record.value() == null)) {
```

Comment:
Same thing here - the root problem is that null can be received by the sink task. And what you have the sink task doing is good - log the condition, and then move on. 

If this class gets a null object, I think a null pointer exception is the way to go. Otherwise, something else will likely blow up later when it tries to handle a "null" AbstractWriteHandle, and it'll be more difficult to debug vs if a null pointer is thrown right away here.